parsers/to-theme-ui/README.md

```diff
@@ -372,7 +372,7 @@ const theme = {
       fontFamily: 'Roboto-Regular',
       lineHeight: '20px',
```

Comment:
This one should be a variant too

Reply:
Both of them? Or only `fontFamily` or `lineHeight`?

Reply:
Fixed ✅